.drone.yml

```diff
@@ -1,21 +1,35 @@
+workspace:
+  base: /go
+
 pipeline:
   test:
     image: golang:1.6
     environment:
       - CGO_ENABLED=0
-      - GO15VENDOREXPERIMENT=1
-      - GOPATH=/drone
     commands:
+      # Disabled: main.go:99: cli.StringSlice composite literal uses unkeyed fields
+      # - go vet
       - go test -cover -coverprofile=coverage.out
-      - go build -ldflags "-s -w -X main.build=$DRONE_BUILD_NUMBER"
-  docker:
-    storage_driver: vfs
+      - go build -ldflags "-s -w -X main.build=$DRONE_BUILD_NUMBER" -a -tags netgo
+
+  latest:
+    image: docker
+    storage_driver: overlay
     repo: plugins/docker
-    tag: [ "latest", "1.0", "1" ]
+    tags: [ "latest", "1.0", "1" ]
     when:
       branch: master
       event: push
 
+  develop:
+    image: docker
```

Comment:
Do we also want to post different becomes per build, e.g. put the build number into it?